modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/content/json/UpdateContentJson.java

```diff
@@ -59,10 +60,14 @@
         final PropertyTree contentData = PropertyTreeJson.fromJson( propertyArrayJsonList );
         final ExtraDatas extraDatas = parseExtradata( extraDataJsonList );
 
+        final PrincipalKey modifier = ContextAccessor.current().getAuthInfo().getUser() != null
+            ? ContextAccessor.current().getAuthInfo().getUser().getKey()
+            : PrincipalKey.ofAnonymous();
+
         this.updateContentParams = new UpdateContentParams().
             requireValid( Boolean.parseBoolean( requireValid ) ).
             contentId( ContentId.from( contentId ) ).
-            modifier( PrincipalKey.ofAnonymous() ).
+            modifier( modifier ).
```

Comment:
Do we use this field at all? We should not, probably as we should not trust the caller.